components/MerkleTree/Visualisation/getLinkComponent.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentType } from 'react';
+import {
+  LinkVertical,
+  LinkVerticalStep,
+  LinkVerticalLine,
+} from '@visx/shape';
+
```

Comment:
Probably move interface out to a type declaration here for function.

Reply:
FIxed